dietpi/dietpi-config

```diff
@@ -367,13 +361,13 @@
 
 			if (( $G_WHIP_RETURNED_VALUE == 15 )); then
 
-				for i in "${!overscan_options[@]}"
+				for i in "${overscan_options[@]}"
 				do
 
-					G_WHIP_DEFAULT_ITEM="$i"
-					if G_WHIP_INPUTBOX "Please enter a value (pixel count) for ${overscan_options[$i]}.\n - EG: 16"; then
+					G_WHIP_DEFAULT_ITEM=${!i}
```

Comment:
Expends `$i` and then the value of variable `$i`:

EG:
- `$i == overscan_left`
- `$overscan_left == 15`
- => `${!i} == 15`